Description: compatibility with the unicode-data package
 Adjust the script used for unicode header file generation for compatibility
 with the Debian unicode-data package.
Author: Steve Langasek <vorlon@debian.org>
Bug-Debian: http://bugs.debian.org/603405
Index: gucharmap-3.4.1.1/gucharmap/gen-guch-unicode-tables.pl
===================================================================
--- gucharmap-3.4.1.1.orig/gucharmap/gen-guch-unicode-tables.pl
+++ gucharmap-3.4.1.1/gucharmap/gen-guch-unicode-tables.pl
@@ -12,7 +12,7 @@
 #  - unicode-versions.h
 #
 # usage: ./gen-guch-unicode-tables.pl UNICODE-VERSION DIRECTORY
-# where DIRECTORY contains UnicodeData.txt Unihan.zip NamesList.txt Blocks.txt Scripts.txt
+# where DIRECTORY contains UnicodeData.txt Unihan_Readings.txt.bz2 NamesList.txt Blocks.txt Scripts.txt
 #
 # NOTE! Some code copied from glib/glib/gen-unicode-tables.pl; keep in sync!
 
@@ -20,7 +20,7 @@
 use vars ('$UNZIP', '$ICONV');
 
 # if these things aren't in your path you can put full paths to them here
-$UNZIP = 'unzip';
+$UNZIP = 'bunzip2';
 $ICONV = 'iconv';
 
 sub process_unicode_data_txt ($);
@@ -40,7 +40,7 @@
 Usage: $0 UNICODE-VERSION DIRECTORY
 
 DIRECTORY should contain the following Unicode data files:
-UnicodeData.txt Unihan.zip NamesList.txt Blocks.txt Scripts.txt
+UnicodeData.txt Unihan_Readings.txt.bz2 NamesList.txt Blocks.txt Scripts.txt
 
 which can be found at http://www.unicode.org/Public/UNIDATA/
 
@@ -55,7 +55,7 @@
 for my $f (readdir ($dir))
 {
     $unicodedata_txt = "$d/$f" if ($f =~ /UnicodeData.*\.txt/);
-    $unihan_zip = "$d/$f" if ($f =~ /Unihan.*\.zip/);
+    $unihan_zip = "$d/$f" if ($f =~ /Unihan_Readings\.txt\.bz2/);
     $nameslist_txt = "$d/$f" if ($f =~ /NamesList.*\.txt/);
     $blocks_txt = "$d/$f" if ($f =~ /Blocks.*\.txt/);
     $scripts_txt = "$d/$f" if ($f =~ /Scripts.*\.txt/);
@@ -63,7 +63,7 @@
 }
 
 defined $unicodedata_txt or die "Did not find $d/UnicodeData.txt";
-defined $unihan_zip or die "Did not find $d/Unihan.zip";
+defined $unihan_zip or die "Did not find $d/Unihan_Readings.txt.bz2";
 defined $nameslist_txt or die "Did not find $d/NamesList.txt";
 defined $blocks_txt or die "Did not find $d/Blocks.txt";
 defined $scripts_txt or die "Did not find $d/Scripts.txt";
